extra_tests/snippets/bytearray.py

```diff
@@ -720,3 +721,15 @@
 assert a.__ne__(b) is False
 b = bytearray(b'my bytearray')
 assert a.__ne__(b) is True
+
+# pickle
+a = bytearray(b'\xffab\x80\0\0\370\0\0')
+# TODO: Different output when pickle dumping a str if protocol is 0
+# CPython's output is like:
+# b'c__builtin__\nbytearray\np0\n(c_codecs\nencode\np1\n(V\xffab\x80\\u0000\\u0000\xf8\\u0000\\u0000\np2\nVlatin1\np3\ntp4\nRp5\ntp6\nRp7\n.'
+# We do have fully Interoperability between our output and CPython's
+assert pickle.dumps(a, 0) == b'c__builtin__\nbytearray\np0\n(c_codecs\nencode\np1\n(V\xffab\x80\x00\x00\xf8\x00\x00\np2\nVlatin1\np3\ntp4\nRp5\ntp6\nRp7\n.'
```

Comment:
CPython use double backslash but our str use one for '\uxxxx', and for '\0' CPython use '\\u0000' but ours is '\x00'.
But interoperability seem works both side.